### ISSUE TITLE
Schedule next meeting (February)

### ISSUE THREAD

**Issue by dontlaugh** [Raku/infrastructure-doc#5]

**Comment by lizmat** [comment]
I don't think we can wait another month to get the new documentation site live.

Documentation is not getting updated.  People are losing interest.  This is **NOT** good.

:-(
